MAIN-3359

timmyquivy opened the pull request.

timmyquivy committed:
MAIN-3359

Making the user parameters more consistent.

timmyquivy committed:
MAIN-3359

Grunny commented:
It's OK to rename the URL parameter, but leave the variables as `wgAllowUserJs` and `wgAllowUserCss` as they are the core globals and are used elsewhere. Also, just a note that we now have [Pull Request guidelines](https://github.com/Wikia/guidelines/blob/master/Process/PullRequests.md), so make sure to include a descriptive title and a short description in all PRs.

timmyquivy commented:
Add I grepped the variables to see if they were used elsewhere and returned
nothing. I'll go back to the drawing board in the AM.

On Wed, Feb 4, 2015 at 11:14 PM, Grunny <notifications@github.com> wrote:

> It's OK to rename the URL parameter, but leave the variables as
> wgAllowUserJs and wgAllowUserCss as they are the core globals and are
> used elsewhere. Also, just a note that we now have Pull Request guidelines
> <https://github.com/Wikia/guidelines/blob/master/Process/PullRequests.md>,
> so make sure to include a descriptive title and a short description in all
> PRs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/app/pull/6175#issuecomment-72991550>.
>



-- 
*TIM QUIEVRYN*
DIRECTOR OF TECHNICAL SUPPORT


*Videogames  */  *Entertainment  */
*Lifestyle*
......................................................................
500 3rd Street, San Francisco, CA 94107
*E *timq@wikia-inc.com
*C* 704-840-4246
*WEB* WIKIA <http://wikia.com/> | FACEBOOK <http://www.facebook.com/wikia>
 | TWITTER <http://twitter.com/wikia>

Grunny commented:
They're at least used in [Preferences](https://github.com/Wikia/app/blob/dev/includes/Preferences.php#L489), [OutputPage](https://github.com/Wikia/app/blob/dev/includes/OutputPage.php#L2899), and our [RailController](https://github.com/Wikia/app/blob/dev/skins/oasis/modules/RailController.class.php#L72) for Oasis. But since the issue is just a user facing URL parameter issue, I'd just cut this down to a two line change that just switches the `$request->getBool( 'allowuserjs', $wgAllowUserJs )` to `$request->getBool( 'useuserjs', $wgAllowUserJs )` and the same for CSS. Changing the variable names would probably be confusing if any engineers were looking at any docs on how to dis/enable use CSS and JS, i.e. https://www.mediawiki.org/wiki/Manual:$wgAllowUserJs :)

timmyquivy committed:
MAIN-4216

Adding an extra line to make it more transparent in ZenDesk that the user is not logged in.

Grunny commented:
Closing, going to create a new PR, some extra stuff got in this branch anyway.
